plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/AdditionalPayments.php

```diff
@@ -137,10 +137,15 @@ private function has_enabled_non_psp_payment_suggestion(): bool {
 	 */
 	private function get_payment_providers(): array {
 		try {
+			/**
+			 * The Payments Settings [page] service.
+			 *
+			 * @var SettingsPaymentsService $settings_payments_service
+			 */
 			$settings_payments_service = wc_get_container()->get( SettingsPaymentsService::class );
 
-			$providers = $settings_payments_service->get_payment_providers( $settings_payments_service->get_country() );
-		} catch ( \Exception $e ) {
+			$providers = $settings_payments_service->get_payment_providers( $settings_payments_service->get_country(), false );
```

Comment:
Not setting `false` is an oversight from [a previous PR](https://github.com/woocommerce/woocommerce/pull/58580) due to the lack of IDE properly identifying the instance type.